### PR TITLE
dev-libs/sway: version bump to 0.9

### PR DESCRIPTION
**Pull request by Hummer12007**
Depends on #2040

### CODE REVIEW

File: dev-libs/sway/sway-0.9.ebuild [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+RDEPEND=">=dev-libs/wlc-0.0.5[systemd=]
+		dev-libs/json-c
+		dev-libs/libpcre
+		dev-libs/libinput
+		x11-libs/libxkbcommon
+		dev-libs/wayland
+		x11-libs/pango
+		x11-libs/cairo
+		swaylock? ( virtual/pam )
+		gdk-pixbuf? ( x11-libs/gdk-pixbuf[jpeg] )"
```
Comment (SoapGentoo) [inline dev-libs/sway/sway-0.9.ebuild]: too much indenting

File: dev-libs/sway/sway-0.9.ebuild [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+DEPEND="${RDEPEND}
+	virtual/pkgconfig
+		app-text/asciidoc"
```
Comment (SoapGentoo) [inline dev-libs/sway/sway-0.9.ebuild]: same here

File: dev-libs/sway/sway-0.9.ebuild [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+}
+
+src_install() {
+	cmake-utils_src_install
+
+	use !systemd && fperms u+s /usr/bin/sway
+}
+
+pkg_postinst() {
+	if use swaygrab
```
Comment (SoapGentoo) [inline dev-libs/sway/sway-0.9.ebuild]: prefer `if use swaygrab; then` syntax

**Comment by Hummer12007** [comment]
@SoapGentoo the same issues were in the former versions, so I'll leave like that it unless someone from the team asks.

**Comment by monsieurp** [comment]
Also, pls fix the issue caused by the PR (above QA warning).

**Pull request closed without merging**